openadmet/models/anvil/workflow.py

```diff
@@ -292,6 +325,7 @@ class AnvilWorkflowBase(BaseModel):
     split: SplitterBase
     feat: FeaturizerBase
     model: ModelBase
+    ensemble: EnsembleBase | None
```

Comment:
```suggestion
    ensemble: Optional[EnsembleBase] = None
```

I have found this pattern seems to work better with Pydantic.

Reply:
From [here](https://stackoverflow.com/a/78920456): "the `type | None = None` option is now recommended for projects only using Python 3.10+"